### PR TITLE
Create executive board page

### PR DESCRIPTION
**Pull request by ksvkabra**

### CODE REVIEW

**Comment by amritsrivastava** [comment]
@ksvkabra why the upper `borderRadius` of name and designation div are curved and also, why there is a minute margin of 1px between this div and the outer image ?

File: src/pages/executive-board.js [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+    width: theme.spacing(6),
+    height: theme.spacing(6),
+  },
+}))
+
+function Something(props) {
+  const classes = useStyles(props)
+  const theme = useTheme()
+
+  const committees = ['UNSC', 'UNODC', 'DISEC', 'UNCSW', 'WHO', 'AIPPM']
```
Comment (amritsrivastava) [inline src/pages/executive-board.js]: why you are not importing this data from data folder ??

File: src/pages/executive-board.js [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+                <Typography style={{ color: '#000a2a' }} variant='h4'>
+                  {name}
+                </Typography>
+                <img
+                  src='/images/line.png'
+                  className='mx-auto'
+                  alt='---------------------'
+                />
+              </Grid>
+              {[1, 2].map(value => (
```
Comment (amritsrivastava) [inline src/pages/executive-board.js]: create a sample recorder for EB in the data file and use that, 
this is very irrelevant and unprofessional way of doing work

**Review by amritsrivastava — CHANGES_REQUESTED** [review]

File: src/pages/executive-board.js (hunk 1 above)
Comment (ksvkabra) [reply src/pages/executive-board.js]: Will correct this in next pr.

**Pull request merged**